lib/TPP/ToBlockLayoutAndBack.cpp

```diff
@@ -423,6 +409,12 @@ mlir::linalgx::packMatmulOp(RewriterBase &rewriter, linalg::MatmulOp matmulOp,
   return packMatmulOpImpl(rewriter, matmulOp, tiles);
 }
 
+//===----------------------------------------------------------------------===//
+// MatmulOp (VNNI packing)
+//===----------------------------------------------------------------------===//
+// Original layout: [IB][JB][ib][jb] += [IB][KB][ib][kb] * [JB][KB][kb][jb]
```

Comment:
Nit: can you try to align the two layouts for easier visual comparison? Maybe line break both?